Project_2/djangoProject/cars/models.py

```diff
@@ -11,39 +11,39 @@ class Car(models.Model):
                                            RegexValidator(r'^[A-Z0-9]*$',
                                                           'The car VIN has only numbers and capital letters!')])
     date_added = models.DateTimeField('date added', auto_now_add=True)
-    title = models.CharField(max_length=100, null=True)
-    description = models.TextField(max_length=500, null=True)
+    title = models.CharField(max_length=100, default='')
+    description = models.TextField(max_length=500, default='')
     year = models.PositiveSmallIntegerField(validators=[MaxValueValidator(timezone.now().year),
                                                         MinValueValidator(1900)],
                                             null=True)
-    car_maker = models.CharField(max_length=100)
-    car_model = models.CharField(max_length=100)
-    car_trim = models.CharField(max_length=100, null=True)
+    car_maker = models.CharField(max_length=100, default='')
```

Comment:
One thing about these two field and also `car_vin`. They don't need the "car" prefix, as the model in named Car and it's clear that they belong to the car and not something else. I would suggest to keep them simple as "vin", "make", "model".